dev-haskell/fay/fay-0.24.2.0.ebuild

```diff
@@ -16,9 +16,11 @@ LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE="test"
-# See [issue upstream][1] for resolving circular dependency
-# [1]: https://github.com/faylang/fay/issues/473
-RESTRICT="test"
+
+# Tests currently fail due to a missing `fay-base` dependency. This was taken
+# out by upstream to resolve a circular-dependency issue. See issue 473 upstream
+# for more information and steps being taken to resolve this.
+# RESTRICT="test"
```

Comment:
I'm  still confused. The comment says tests fail, but `RESTRICT=test` is commented out. Should it be?

Reply:
🤦‍♂️ I think I was just staring at this too much, it should still be restricted.